.goreleaser.yml

```diff
@@ -36,10 +36,47 @@ dockers:
 - goos: linux
   goarch: amd64
   image_templates:
-  - "percona/mongodb_exporter:{{.Major}}.{{.Minor}}"
-  - "percona/mongodb_exporter:{{.Version}}"
+  - "docker.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64"
+  - "docker.io/percona/mongodb_exporter:{{.Version}}-amd64"
+  - "ghcr.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64"
+  - "ghcr.io/percona/mongodb_exporter:{{.Version}}-amd64"
+  dockerfile: Dockerfile
+  build_flag_templates:
+    - "--pull"
+    - "--platform=linux/amd64"
+- goos: linux
+  goarch: arm64
+  image_templates:
+  - "docker.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8"
+  - "docker.io/percona/mongodb_exporter:{{.Version}}-arm64v8"
+  - "ghcr.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8"
+  - "ghcr.io/percona/mongodb_exporter:{{.Version}}-arm64v8"
   dockerfile: Dockerfile
   use: buildx
+  build_flag_templates:
+    - "--pull"
```

Comment:
@nikita-b / @BupycHuk Do we need pull?

Reply:
I'm not sure what does it here, but probably it pulls the latest version of the image to not use the one in a cache. So I think it's ok.
@nikita-b your thoughts